src/base/bittorrent/customstorage.h

```diff
@@ -49,6 +49,8 @@ class CustomStorage final : public lt::default_storage
     lt::status_t move_storage(const std::string &savePath, lt::move_flags_t flags, lt::storage_error &ec) override;
 
 private:
+    void handleCompleteFiles(const QString &savePath);
```

Comment:
Seems like a const function to me?

Reply:
Don't forget that this `const` refers to the method interface. You shouldn't declare a method as `const` just because its current implementation doesn't change the object.
This one doesn't seem like a const method to me.